components/scream/src/interface/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ set(ATM_SRC
   dead_mct_mod.F90
   dead_data_mod.F90
   dead_mod.F90
+  scream_scorpio_interface.F90
```

Comment:
I agree, I would anticipate it be a tool we want to use in stand-alone as well.  For the time being we would still need to put in CMake directives that skipped the PIO stuff in non-CIME builds or take the time to get SCREAM building all of PIO in stand-alone mode.  Otherwise the builds will break.  Would it make sense to keep the scorpio code in `interface` for now so that it is still associated only with FSCREAM-SA builds and put together a complete PR which moves all of the files to a new location in `share` and does the CMake work to make that feasible?

Reply:
I'd vote for renaming interface => mct_coupling to be more clear what it means. We have interfaces in front of all of our parameterizations so overuse of the word will be confusing.

Reply:
@PeterCaldwell do you mean the whole folder? Seems reasonable to me. Interface is indeed an overloaded name.

Reply:
Yeah, I meant just rename the folder.

Reply:
@AaronDonahue up to you if you want to do that in this PR, or leave it for the next mct one

Reply:
Might as well do it now.  I'll make the change and push.